### PR TITLE
Master

### PR DESCRIPTION
**Pull request by M-Khairy-B**

### CODE REVIEW

File: src/template/partials/header.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<nav class="navbar navbar-expand-lg bg-body-tertiary navbar-main">
+    <div class="container p-0">
```
Comment (mukhtar-ali) [inline src/template/partials/header.html]: Missing header tag!

Comment (mukhtar-ali) [inline src/template/partials/header.html]: We agreed that we never do styles to container, row and columns. 
Why are you removing the padding here?

Comment (mukhtar-ali) [inline src/template/partials/header.html]: Alignment issue

File: src/template/partials/header.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                    <a class="nav-link active" aria-current="page" href="#contact"
+                        >Contact</a
+                    >
+                </li>
+            </ul>
+            <div class="d-lg-flex btn-main" role="search">
+                <div class="d-flex justify-content-between align-items-center switch-theme">
+                    <span class="d-lg-none">Switch Theme</span>
+                    <button class="border-0 icon-mode"><img src="assets/images/icon-light-mode.svg" alt=""></button>
+                </div>
```
Comment (mukhtar-ali) [inline src/template/partials/header.html]: Space between the icon and button not like the design, that's due to the icon has extra space inside.

File: src/pages/index.html [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+                            a focus on creating (and occasionally designing)
+                            exceptional digital experiences that are fast,
+                            accessible, visually appealing, and responsive. Even
+                            though I have been creating web applications for
+                            over 7 years, I still love it as if it was something
+                            new.
+                        </p>
+                        <div class="info-person">
+                            <div class="location d-flex align-items-center">
+                                <i class="fa-solid fa-location-dot"></i>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: Icon not like the design!!

File: src/pages/index.html [hunk 4]
```diff
@@ -0,0 +1,501 @@
...
+                                <span class="name-main">Ahmedabad, India</span>
+                            </div>
+                            <div class="new-project d-flex align-items-center">
+                                <div class="circle-main"></div>
+                                <span class="available"
+                                    >Available for new projects</span
+                                >
+                            </div>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: Both are not like the design in terms of spacing between icons and text

File: src/pages/index.html [hunk 5]
```diff
@@ -0,0 +1,501 @@
...
+                                <a href="">
+                                    <i class="fa-brands fa-twitter"></i>
+                                </a>
+                            </div>
+                            <div class="box">
+                                <a href="">
+                                    <i class="fa-brands fa-figma"></i>
+                                </a>
+                            </div>
+                        </div>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: Icons are not like the design

File: src/pages/index.html [hunk 6]
```diff
@@ -0,0 +1,501 @@
...
+                <div class="col-12 col-lg-4 p-0">
+                    <div class="wrapper-pic">
+                        <div class="pic-main">
+                            <img
+                                class="w-100"
+                                src="assets/images/Pic.jpg"
+                                alt=""
+                                srcset=""
+                            />
+                            <div class="wrapper-gray"></div>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: It's best practices to make the gray box with after or before

File: src/pages/index.html [hunk 7]
```diff
@@ -0,0 +1,501 @@
...
+                        </div>
+                    </div>
+                </div>
+                <div class="col-12 col-lg-6 offset-lg-1">
+                    <div class="wrapper-about-me">
+                        <h2>Curious about me? Here you have it:</h2>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: This should be h3 instead as you have h2 already inside the section

File: src/template/component/card.html [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+                        <li>{{ data.experienceFour }}</li>
+                    {% endif %}
+                </ul>
+            </div>
+        </div>
+        <div class="date-experience">
+            <span>{{ data.date }}</span>
+        </div>
+    </div>
+</div>
```
Comment (mukhtar-ali) [inline src/template/component/card.html:24]: You should keep the grid details outside of the component, Imagine using the same component in a different place with another grid setup.

File: src/template/component/card.html [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+            <h4>{{ data.title }}</h4>
+            <div class="details-experience">
+                <ul>
+                    <li>{{ data.experience }}</li>
+                    <li>{{ data.experienceTwo }}</li>
+                    <li>{{ data.experienceThree }}</li>
+                    {% if data.experienceFour %}
+                        <li>{{ data.experienceFour }}</li>
+                    {% endif %}
+                </ul>
```
Comment (mukhtar-ali) [inline src/template/component/card.html:17]: Not aligned like the design

File: src/template/component/card.html [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+            </div>
+        </div>
+        <div class="date-experience">
+            <span>{{ data.date }}</span>
+        </div>
+    </div>
```
Comment (mukhtar-ali) [inline src/template/component/card.html:23]: Missing background color

File: src/pages/index.html [hunk 11]
```diff
@@ -0,0 +1,501 @@
...
+                                    adipiscing elit.
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="date-experience">
+                        <span>Nov 2021 - Present</span>
+                    </div>
+                </div>
+            </div>
```
Comment (mukhtar-ali) [inline src/pages/index.html:220]: Why you didn't use the macro you created?

File: src/pages/index.html [hunk 12]
```diff
@@ -0,0 +1,501 @@
...
+    <div class="container">
+        <div class="wrapper-work">
+            <div class="row">
+                <div class="col-12 col-lg-6">
+                    <div class="img-work ">
+                        <img
+                            class="w-100"
+                            src="assets/images/Picture.png"
+                            alt=""
+                        />
```
Comment (mukhtar-ali) [inline src/pages/index.html:293]: Not like the design

File: src/pages/index.html [hunk 13]
```diff
@@ -0,0 +1,501 @@
...
+                    </div>
+                </div>
+                <div class="col-12 col-lg-6">
+                    <div class="details-project">
+                        <div class="project-title">
+                            <h4>Fiskil</h4>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: h3

File: src/pages/index.html [hunk 14]
```diff
@@ -0,0 +1,501 @@
...
+                            <span>Tailwindcss</span>
+                            <span>Figma</span>
+                            <span>Cypress</span>
+                            <span class="mb-0">Storybook</span>
+                            <span class="mb-0">Git</span>
+                        </div>
+                        <div class="icon-button">
+                            <a href="#"
+                                ><img
+                                    src="assets/images/Icon Button.svg"
```
Comment (mukhtar-ali) [inline src/pages/index.html]: use small letter in all of your naming convention

File: src/pages/index.html [hunk 15]
```diff
@@ -0,0 +1,501 @@
...
+                    <div class="img-work ">
+                        <img
+                            class="w-100"
+                            src="assets/images/Picture.png"
+                            alt=""
+                        />
+                    </div>
+                </div>
+                <div class="col-12 col-lg-6">
+                    <div class="details-project">
```
Comment (mukhtar-ali) [inline src/pages/index.html]: Missing box shadow

File: src/pages/index.html [hunk 16]
```diff
@@ -0,0 +1,501 @@
...
+            <p class="main-p-testimonials">
+                Here is a quick summary of my most recent experiences:
+            </p>
+        </div>
+    </div>
+    <div class="container">
+        <div class="wrapper-testimonials">
+            <div class="row">
+                <div class="col-12 col-xl-4">
+                    <div class="card-testimonials">
```
Comment (mukhtar-ali) [inline src/pages/index.html:439]: background color is missing

Comment (mukhtar-ali) [inline src/pages/index.html:439]: insider space not like the design.

File: src/template/partials/footer.html [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
+                        <div class="main-phone">
+                            <span class="phone"
+                                ><i class="fa-solid fa-phone"></i
+                            ></span>
+                            <span class="number-phone"><a class="number-phone" href="#">+91 8980500565</a></span>
+                            <img
+                                src="assets/images/Icon Button-2.svg"
+                                alt=""
+                            />
+                        </div>
```
Comment (mukhtar-ali) [inline src/template/partials/footer.html]: Not like the design

File: src/template/partials/footer.html [hunk 18]
```diff
@@ -0,0 +1,62 @@
...
+                    </div>
+                    <div class="find-me text-center">
+                        <span class="platforms">You may also find me on these platforms!</span>
+                        <div class="find-me-link">
+                            <a href="#"> <i class="fa-brands fa-github"></i> </a>
+                            <a href="#">
+                                <i class="fa-brands fa-twitter"></i>
+                            </a>
+                            <a href="#"> <i class="fa-brands fa-figma"></i> </a>
+                        </div>
```
Comment (mukhtar-ali) [inline src/template/partials/footer.html]: Icons?

**Review by mukhtar-ali — CHANGES_REQUESTED** [review]

File: src/template/partials/header.html (hunk 1 above)
Comment (M-Khairy-B) [reply src/template/partials/header.html]: Done

File: src/template/partials/header.html (hunk 2 above)
Comment (M-Khairy-B) [reply src/template/partials/header.html]: Done

File: src/pages/index.html (hunk 3 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: Done

File: src/pages/index.html (hunk 4 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: Done

File: src/pages/index.html (hunk 5 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: Done

File: src/pages/index.html (hunk 6 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: Done

File: src/template/partials/header.html (hunk 1 above)
Comment (M-Khairy-B) [reply src/template/partials/header.html]: Done

Comment (M-Khairy-B) [reply src/template/partials/header.html]: Done

File: src/pages/index.html (hunk 7 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: Done

File: src/template/component/card.html (hunk 10 above)
Comment (M-Khairy-B) [reply src/template/component/card.html:23]: Done

File: src/template/component/card.html (hunk 8 above)
Comment (M-Khairy-B) [reply src/template/component/card.html:24]: ok

File: src/template/component/card.html (hunk 9 above)
Comment (M-Khairy-B) [reply src/template/component/card.html:17]: Done

File: src/pages/index.html (hunk 12 above)
Comment (M-Khairy-B) [reply src/pages/index.html:293]: Done

File: src/pages/index.html (hunk 13 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: Done

File: src/pages/index.html (hunk 15 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: Done

File: src/pages/index.html (hunk 16 above)
Comment (M-Khairy-B) [reply src/pages/index.html:439]: Done

Comment (M-Khairy-B) [reply src/pages/index.html:439]: Done

File: src/template/partials/footer.html (hunk 18 above)
Comment (M-Khairy-B) [reply src/template/partials/footer.html]: Done

File: src/template/partials/footer.html (hunk 17 above)
Comment (M-Khairy-B) [reply src/template/partials/footer.html]: Done

File: src/template/partials/header.html (hunk 2 above)
Comment (Ahmed-Hassan-96) [reply src/template/partials/header.html]: margin to parent not to button

File: src/pages/index.html (hunk 3 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html]: not done 
Where is the icon ?

File: src/pages/index.html (hunk 4 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html]: not done
Same

File: src/pages/index.html (hunk 5 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html]: Not done

File: src/pages/index.html (hunk 6 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html]: not done where is the grey background

File: src/template/component/card.html (hunk 8 above)
Comment (Ahmed-Hassan-96) [reply src/template/component/card.html:24]: Not done

File: src/template/component/card.html (hunk 9 above)
Comment (Ahmed-Hassan-96) [reply src/template/component/card.html:17]: not done

File: src/pages/index.html (hunk 11 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html:220]: not done

File: src/pages/index.html (hunk 12 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html:293]: not done

File: src/pages/index.html (hunk 15 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html]: not done

File: src/pages/index.html (hunk 16 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html:439]: not done

File: src/template/partials/footer.html (hunk 17 above)
Comment (Ahmed-Hassan-96) [reply src/template/partials/footer.html]: not done

File: src/pages/index.html (hunk 3 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: I was wrong src

File: src/pages/index.html (hunk 5 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: Done

File: src/pages/index.html (hunk 3 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: Done

File: src/pages/index.html (hunk 6 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: Done 
وربنا معموله

File: src/template/component/card.html (hunk 8 above)
Comment (M-Khairy-B) [reply src/template/component/card.html:24]: I don't do it macron

File: src/pages/index.html (hunk 11 above)
Comment (M-Khairy-B) [reply src/pages/index.html:220]: I don't do it macron
i said Mukhtar

File: src/pages/index.html (hunk 4 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: I swear to God 
same Almost a figma

File: src/template/partials/footer.html (hunk 17 above)
Comment (M-Khairy-B) [reply src/template/partials/footer.html]: I want to know the reason ?

File: src/pages/index.html (hunk 15 above)
Comment (M-Khairy-B) [reply src/pages/index.html]: Done

File: src/pages/index.html (hunk 12 above)
Comment (M-Khairy-B) [reply src/pages/index.html:293]: the img make box shadow

File: src/template/partials/header.html (hunk 2 above)
Comment (M-Khairy-B) [reply src/template/partials/header.html]: Done

**Pull request closed without merging**